### ISSUE TITLE
Incorrect comparison

### ISSUE THREAD

**Issue by joeri** [shime/node-vs-ruby-io#2]
In Ruby reading the file `image = Vips::Image.new_from_file("bridge.jpg")` takes place before starting the timer.

In JS the reading takes place after reading the file.

Note that it's possible that `Vips::Image.new_from_file` is lazy and doesn't do much, but it would be better either way to move it inside to make the comparison valid.

**Comment by janko** [comment]
@joeri Good point, updated in https://github.com/shime/node-vs-ruby-io/commit/94dcfc154a170a856371d5d6267cb2f5c391fdc1.

**Comment by jcupitt** [comment]
Yes, libvips is lazy and won't do the decompress until the actual pixels are read. You're right, it's clearer to move it inside the timer.
